test/handler/test_reek_handler.vader

```diff
@@ -0,0 +1,67 @@
+Execute(The reek handler should parse JSON correctly):
+    runtime ale_linters/ruby/reek.vim
+
+    let g:ale_ruby_reek_show_context = 1
+    let g:ale_ruby_reek_show_wiki_link = 0
```

Comment:
You should split each of these checks into separate `Execute` blocks. That way, the tests can fail later for 1 out of 4 cases, and we can spot which one is broken.